src/main/kotlin/com/strumenta/kolasu/model/Processing.kt

```diff
@@ -162,19 +162,19 @@ val Node.children: List<Node>
     }
 
 // TODO reimplement using transformChildren
```

Comment:
We should also update this comment now, I think

Reply:
I'll do that in the next PR.